activities/ActivityTemplateVue.activity/js/components/toolbar.js

```diff
@@ -32,6 +32,16 @@ Vue.component('toolbar-item', {
 				}
 			});
 		}
+
+		if(vm.id == 'network-button') {
+			requirejs(["sugar-web/graphics/presencepalette"], function(presencepalette) {
+				var presencePalette = new presencepalette.PresencePalette(document.getElementById(vm.id), undefined);
+				presencePalette.addEventListener('shared', function () {
+					presencePalette.popDown();
+					vm.$root.$refs.presence.onShared();
+				});
+			});
```

Comment:
You don't need to specify the source code. Just use the `paletteFile`, `paletteClass` and `paletteEvent` in `toolbar-item`. See for example [here](https://github.com/llaske/sugarizer/blob/dev/activities/Calligra.activity/js/toolbar.js#L51).

Reply:
The attributes `paletteFile`, `'paletteClass`, etc are used to create a normal Palette. The presence palette has a different library which needs to be called I believe. Can the presence palette be created with `new palette` instead of `new presencepalette`?

Reply:
It's a bit tricky but the `paletteClass` is use to create the object with the right JavaScript class. See the source code of this call [here](https://github.com/llaske/sugarizer/blob/dev/activities/Calligra.activity/js/toolbar.js#L32).
So it will works if you're using `'presencepalette'` value for `paletteClass` property.
In a same way if you're using `'shared'` value for `paletteEvent` property, you will be able to use `v-on:shared` syntax to handle the event.

Reply:
Wow that's clever! I'll test it soon.

Reply:
Added in c428dd8